ts/main.ts

```diff
@@ -42,7 +50,18 @@ function submitHandler(event: Event): void {
   };
   data.entries.unshift(newEntry);
   data.nextEntryId++;
-  storeData();
+  const $liToAppend = renderEntry(data.entries[0]);
+  if (!$ulForEntries) throw new Error('$ulForEntries not found!');
+  $ulForEntries.prepend($liToAppend);
+  viewSwap('entries');
+  storeData(); // Is after viewSwap because viewSwap changes data.view
+  if (!$noEntries) throw new Error('$noEntries not found!');
+  if (
+    $ulForEntries.childNodes.length > 1 &&
+    !$noEntries.classList.contains('hidden')
+  ) {
```

Comment:
Right here you write this condition, you also write the same condition on lines 124-127. Instead of repeating the condition here, add it into your `toggleNoEntries` function so that these lines are only written once.